app/views/files/index.html.erb

```diff
@@ -16,7 +16,9 @@
           </section>
 
           <section class="file-preview">
-            <% if attachment.image? %>
+            <% if present(attachment).thumbnail(:thumb).present? %>
+              <div class="image-preview" style="background-image: url(<%= present(attachment).thumbnail(:thumb) %>);"></div>
+            <% elsif attachment.image? %>
```

Comment:
Now that we have the processor taking thumbnails you should get rid of checking if its and image and putting the full image URL in, as that could slow down this page a lot if the images are massive.